repair/row_level.hh

```diff
@@ -88,6 +89,7 @@ class repair_service : public seastar::peering_sharded_service<repair_service> {
     sharded<db::system_distributed_keyspace>& _sys_dist_ks;
     sharded<db::system_keyspace>& _sys_ks;
     sharded<db::view::view_update_generator>& _view_update_generator;
+    task_manager::module_ptr _module;
```

Comment:
nit: `_module` is a bit too generic.
I suggest renaming it to `_task_manager_module`.